classes/components/forms/context/PKPContextForm.inc.php

```diff
@@ -53,6 +67,12 @@ public function __construct($action, $locales, $baseUrl, $context) {
 				'groupId' => 'identity',
 				'value' => $context ? $context->getData('acronym') : null,
 			]))
+			->addField(new FieldSelect('country', [
+				'label' => __('manager.setup.selectCountry'),
```

Comment:
Can we make the label Country and in the description describe what it's used for? Like:

> *Country*
> Select the country where this journal is located, or the country of the mailing address for the journal or publisher.